### PR TITLE
Update the explorer links to work with localhost

### PR DESCRIPTION
**Pull request by kenny08gt**
## What

- Update explorer references to work with LocalHost

## Why

- Improve dev experience for rsk port in localhost

## Refs
https://rsklabs.atlassian.net/browse/PP-172

### CODE REVIEW

**Comment by kenny08gt** [comment]
@antomor The update is made to depend on the explorer version. Please check it

File: environments/.env.mainnet [hunk 1]
```diff
@@ -8,3 +8,4 @@ APP_WS_API_ETHERSCAN_TOKEN=FTKI6X11ZGTQ9J1KZ6WIRQ2ZW8EYZ8Y7FC
 APP_CURRENT_NETWORK=mainnet
 SENTRY_DSN=https://674c7230d4594037bc1a9797339aa831@o496053.ingest.sentry.io/5662609
 GTAG_ID=GTM-P9S4QSF
+APP_EXPLORER_VERSION=0.1.0
```
Comment (antomor) [inline environments/.env.mainnet:11]: I don't think we need to change this.

File: environments/.env.rinkeby [hunk 2]
```diff
@@ -8,3 +8,4 @@ APP_WS_API_ETHERSCAN_TOKEN=FTKI6X11ZGTQ9J1KZ6WIRQ2ZW8EYZ8Y7FC
 APP_CURRENT_NETWORK=rinkeby
 SENTRY_DSN=https://674c7230d4594037bc1a9797339aa831@o496053.ingest.sentry.io/5662609
 GTAG_ID=GTM-P9S4QSF
+APP_EXPLORER_VERSION=0.1.0
```
Comment (antomor) [inline environments/.env.rinkeby:11]: I don't think we need to change this.

File: environments/.env.rinkeby-beta [hunk 3]
```diff
@@ -11,3 +11,4 @@ GTAG_ID=GTM-P9S4QSF
 ZK_SPECIAL_API="rinkeby-beta-api.zksync.io"
 ZK_SPECIAL_SCAN="https://rinkeby-beta-scan.zksync.dev"
 ZK_NETWORK="rinkeby-beta"
+APP_EXPLORER_VERSION=0.1.0
```
Comment (antomor) [inline environments/.env.rinkeby-beta:14]: I don't think we need to change this.

File: environments/.env.ropsten [hunk 4]
```diff
@@ -8,3 +8,4 @@ APP_WS_API_ETHERSCAN_TOKEN=FTKI6X11ZGTQ9J1KZ6WIRQ2ZW8EYZ8Y7FC
 APP_CURRENT_NETWORK=ropsten
 SENTRY_DSN=https://674c7230d4594037bc1a9797339aa831@o496053.ingest.sentry.io/5662609
 GTAG_ID=GTM-P9S4QSF
+APP_EXPLORER_VERSION=0.1.0
```
Comment (antomor) [inline environments/.env.ropsten:11]: I don't think we need to change this.

File: environments/.env.ropsten-beta [hunk 5]
```diff
@@ -11,3 +11,4 @@ GTAG_ID=GTM-P9S4QSF
 ZK_SPECIAL_API="ropsten-beta-api.zksync.io"
 ZK_SPECIAL_SCAN="https://ropsten-beta-scan.zksync.dev"
 ZK_NETWORK="ropsten-beta"
+APP_EXPLORER_VERSION=0.1.0
```
Comment (antomor) [inline environments/.env.ropsten-beta:14]: I don't think we need to change this.

**Review by antomor — APPROVED** [review]
LGTM, I'd just rollback the change to the environment files that we are not using at the moment

File: environments/.env.mainnet (hunk 1 above)
Comment (kenny08gt) [reply environments/.env.mainnet:11]: I think we should keep this one, for the moment we deploy to mainnet

File: environments/.env.rinkeby-beta (hunk 3 above)
Comment (kenny08gt) [reply environments/.env.rinkeby-beta]: removed

File: environments/.env.ropsten (hunk 4 above)
Comment (kenny08gt) [reply environments/.env.ropsten]: removed

File: environments/.env.ropsten-beta (hunk 5 above)
Comment (kenny08gt) [reply environments/.env.ropsten-beta:14]: removed

Comment (antomor) [reply environments/.env.ropsten-beta:14]: I cannot find the revert of this change. Could you please double check?

**Pull request merged**